frontend/src/views/CellCards/components/CellCard/index.tsx

```diff
@@ -36,100 +41,127 @@ export default function CellCard(): JSX.Element {
   const sectionRef2 = React.useRef(null);
   const sectionRef3 = React.useRef(null);
 
-  const [skinnyMode, setSkinnyMode] = useState<boolean>(true);
+  const [skinnyMode, setSkinnyMode] = useState<boolean>(false);
```

Comment:
we can now default to `false` and not worry about `window` doesn't exist anymore